tests/unit/Entity/PropertyIdTest.php

```diff
@@ -92,7 +92,7 @@ public function testGetEntityType() {
 
 	public function testSerialize() {
```

Comment:
There is, see https://github.com/wmde/WikibaseDataModel/pull/718/files/8a3e451d87ae29420ff85354f328f80ba1e72363#diff-4d5b4f754035c1d318ef4e6f8c2382a2L101.